pkg/util/olm/operator.go

```diff
@@ -122,7 +123,7 @@ func HasPermissionToInstall(ctx context.Context, client client.Client, namespace
 	}
 
 	if !global {
-		if ok, err := kubernetes.CheckPermission(ctx, client, operatorsv1.GroupName, "operatorgroups", namespace, options.Package, "list"); err != nil {
```

Comment:
Maybe `operatorsv1.GroupVersion.Group`, or use `operators.GroupName` also for the v1alpha1 APIs above for consistency?

Reply:
The `v1alpha1` is using `operators.GroupName` behind the scene: https://github.com/operator-framework/api/blob/v0.13.0/pkg/operators/v1alpha1/register.go#L13 - I'd be inclined to keep this value as it is the value expected by that package and let the API manages. WDYT?

Reply:
Yes, that's sounds good, it's not a big deal.